scale/storage/test/utils.py

```diff
@@ -54,6 +55,22 @@ def create_file(file_name='my_test_file.txt', media_type='text/plain', file_size
         scale_file.countries = countries
         scale_file.save()
     return scale_file
+    
+    
+def create_uningested_file(file_name='my_test_file.txt', file_path=None, contents=None):
+    """Creates a raw file for use in testing list_files broker
```

Comment:
Is creating an actual file necessary for testing? Might be better to just mock any file IO calls that would interact with the file.